includes/class-activitypub.php

```diff
@@ -498,6 +499,25 @@ public static function theme_compat() {
 		}
 	}
 
+	/**
+	 * Pre-get option filter for the Authorized Fetch.
+	 *
+	 * @param string $pre The pre-get option value.
+	 *
+	 * @return string If the constant is defined, return the value, otherwise return the pre-get option value.
+	 */
+	public static function pre_option_activitypub_authorized_fetch( $pre ) {
+		if ( ! \defined( 'ACTIVITYPUB_AUTHORIZED_FETCH' ) ) {
+			return $pre;
+		}
+
+		if ( ACTIVITYPUB_AUTHORIZED_FETCH ) {
+			return '1';
+		}
+
+		return '0';
```

Comment:
```suggestion
		return (bool) ACTIVITYPUB_AUTHORIZED_FETCH;
```
Given the type cast in `use_authorized_fetch` we're only interested in true or false here?

Reply:
If we go ahead with that change, function docs would need to be updated, too

Reply:
I would keep it as is, to be compatible with how `get_option` and `get_user_meta` works...